python/across_api/across/resolve.py

```diff
@@ -0,0 +1,170 @@
+import json
+from typing import Optional, Tuple
+
+import requests
+from fastapi import HTTPException
+
+from ..base.common import ACROSSAPIBase
+from ..base.schema import JobInfo
+from .jobs import check_cache, register_job
+from .schema import ResolveGetSchema, ResolveSchema
+
+ANTARES_URL = "https://api.antares.noirlab.edu/v1/loci"
+
+
+def antares_radec(ztf_id: str) -> Tuple[Optional[float], Optional[float]]:
+    """
+    Query ANTARES API to find RA/Dec of a given ZTF source
+
+    Parameters
+    ----------
+    ztf_id : str
+        ZTF name of source
+
+    Returns
+    -------
+    Tuple[float, float]
+        RA, Dec in ICRS decimal degrees
+    """
+    search_query = json.dumps(
+        {"query": {"bool": {"filter": {"term": {"properties.ztf_object_id": ztf_id}}}}}
+    )
+
+    params = {
+        "sort": "-properties.newest_alert_observation_time",
+        "elasticsearch_query[locus_listing]": search_query,
+    }
+    r = requests.get(ANTARES_URL, params=params)
+
+    if r.status_code == 200:
+        antares_data = json.loads(r.text)
+        ra = antares_data["data"][0]["attributes"]["ra"]
+        dec = antares_data["data"][0]["attributes"]["dec"]
+        return ra, dec
+    else:
+        return None, None
+
+
+def simbad_radec(name: str) -> Tuple[Optional[float], Optional[float]]:
+    """
+    Given a object name, return the Simbad coordinates in degrees.
+
+    Parameters
+    ----------
+    name : str
+        Name of object to search for
+
+    Returns
+    -------
+    tuple
+        RA/Dec in decimal degrees (float or None)
+    """
+    url = "http://simbad.u-strasbg.fr/simbad/sim-script?script="
+    script = 'format object "%IDLIST(1) | %COO(d;A D)\n' + "query id %s" % name
+
+    lines = requests.get(url + script).text.splitlines()
+
+    ddec = None
+    dra = None
+    for line in lines:
+        x = line.split("|")
+        try:
+            name = x[0]
+            numbers = x[1].split(" ")
+            dra = float(numbers[1])
+            ddec = float(numbers[2].strip())
+        except (ValueError, IndexError):
+            pass
```

Comment:
Replace with [stropy.coordinates.SkyCoord.from_name](https://docs.astropy.org/en/stable/api/astropy.coordinates.SkyCoord.html#astropy.coordinates.SkyCoord.from_name).

Reply:
I actually tried to do this before but it crashes for reasons I don't understand.

Reply:
The error:

```  File "...across_api/across/resolve.py", line 133, in get
    skycoord = SkyCoord.from_name(self.name)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "...vendor/astropy/coordinates/sky_coordinate.py", line 2205, in from_name
    icrs_coord = get_icrs_coordinates(name, parse, cache=cache)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "...vendor/astropy/coordinates/name_resolve.py", line 207, in get_icrs_coordinates
    sc = SkyCoord(ra=ra, dec=dec, unit=(u.degree, u.degree), frame="icrs")
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "...vendor/astropy/coordinates/sky_coordinate.py", line 333, in __init__
    frame_cls, frame_kwargs = _get_frame_without_data(args, kwargs)
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "...vendor/astropy/coordinates/sky_coordinate_parsers.py", line 143, in _get_frame_without_data
    frame_cls = _get_frame_class(frame)
                ^^^^^^^^^^^^^^^^^^^^^^^
  File "...vendor/astropy/coordinates/sky_coordinate_parsers.py", line 47, in _get_frame_class
    raise ValueError(
ValueError: Coordinate frame name "icrs" is not a known coordinate frame ([])
```

Reply:
Bug resolved, so 6cfaa75 fixes this.